src/components/PageCard.jsx

```diff
@@ -47,7 +48,7 @@ const PageCard = ({ item, itemIndex }) => {
       return `${url}/editPage/${encodedName}`
     }
     return `/sites/${siteName}/editPage/${encodedName}`
-  }
+  }, [resourceType, resourceRoomName, siteName, encodedName])
```

Comment:
Should `url` also be a dependency?

Reply:
Yes, good catch! Oddly enough there's no eslint for this rven though there is for `useEffect`

Reply:
> Looks great, left a few small comments
> 
> Also, minor nit - the button is slightly off centre: <img alt="Screenshot 2022-06-09 at 10 37 16 AM" width="383" src="https://user-images.githubusercontent.com/22111124/172752430-b340c41d-8376-467f-91c3-e9260a6d4d04.png">

Yeap it is and good catch! I'm aware of this but I'm not addressing it for a few reasons:

1. V1.5 figma actually doesn't have the margins for old cards! The button here is absolutely positioned wrt the card itself so we'd need to change either the position of the button or padding of the card which would be fine if not for...
2. New cards are coming up subsequently and those look fine with the drop-down! As the new cards should go out by this month, I'm not particularly concerned for this.

Tagging @NatMaeTan - any thoughts

Reply:
over slack - no issues